.github/workflows/vercel-deploy-notification.yml

```
@@ -0,0 +1,88 @@
+        uses: actions/github-script@v7
+        id: get-pr
+        with:
+          script: |
+            const branch = context.payload.deployment.ref;
+            const prs = await github.rest.pulls.list({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              head: `${context.repo.owner}:${branch}`,
+              state: 'open'
+            });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)